### ISSUE TITLE
Jade error on `npm start`

### ISSUE THREAD

**Issue by JbIPS** [silexlabs/BackNode#16]
Hi,

Just came back from holidays and catching up with the project! I updated my branch with all your changes and tried the run script `npm start`, but I got a jade error:
```
Error: Jade:41
    39|         #logo
    40|     // backnode app
  > 41|     include src/jade/App.jade
    42|
    43|     // Cloud-Explorer
    44|     iframe(
the "filename" option is required to use "include" with "relative" paths
```
Did I forgot a file to update, or is it something you experience too?

Thank you, and happy new year ;)

**Comment by RonanDrouglazet** [comment]
hmm !! Reaaaaally strange ! The name is here ^^ try to remove your node_module folder or just node_modules/jade folder then npm install again, sometimes work... 

i try it and this work 
> backnode@0.0.1 build:html /Users/drouglazet/Desktop/work/BackNode
> jade < ./src/jade/Index.jade > ./dist/index.html

tell me if it's ok or not ! Thanks :)

**Comment by JbIPS** [comment]
:+1: ! Should have updated my node_modules first, thanks !

**Comment by RonanDrouglazet** [comment]
i have the same issue --> jade 1.9.0 just released and fix the version on ~1.8 fix the problem for me
i will check that later, not critical now !
